fearless/Common/Network/Subquery/SubqueryRewardOperationFactory.swift

```diff
@@ -13,23 +23,39 @@ final class SubqueryRewardOperationFactory {
         self.url = url
     }
 
-    private func prepareQueryForAddress(_ address: String) -> String {
-        """
+    private func prepareQueryForAddress(
+        _ address: String,
+        startTimestamp: Int64?,
+        endTimestamp: Int64?
+    ) -> String {
+        let timestampFilter: String = {
+            guard startTimestamp != nil || endTimestamp != nil else { return "" }
+            var result = "timestamp:{"
+            if let timestamp = startTimestamp {
+                result.append("greaterThanOrEqualTo:\"\(timestamp)\",")
+            }
+            if let timestamp = endTimestamp {
+                result.append("lessThanOrEqualTo:\"\(timestamp)\",")
```

Comment:
I am not sure here, but usually if one case is "greater than or equal" then the other case is strictly "less than"